src/Model/Request/TerminalsBrowseRequest.php

```diff
@@ -16,17 +16,55 @@
  */
 class TerminalsBrowseRequest extends RequestData
 {
-    public function __construct()
+    protected string $merchantCode;
+    protected string $excludeMerchantCode;
+
+    /**
+     * @param string $merchantCode
+     * @param string $excludeMerchantCode
+     */
+    public function __construct(string $merchantCode = '', string $excludeMerchantCode = '')
```

Comment:
If you want to use the setters, then nevermind.
If not really necessary in constructor, we could use php property promotion 😊 
```php
public function __construct(
    public string $merchantCode = '', 
    public string $excludeMerchantCode = ''
)
```

Reply:
Property promotion's nice and clean, but we opted for setters to keep things flexible.